profiles/scraper.py

```diff
@@ -388,13 +415,33 @@ def parse_profile_experiences(self, company, role):
         :return: List of dictionaries containing information about each experience. Return None if no experience.
         """
         experiences = []
-        exps = self.get_multiple_by_xpath_or_none('EXPERIENCES')
+        exps = self.get_multiple_by_xpath_or_none('EXPERIENCES', wait=True)
```

Comment:
why this wait=true?

Reply:
Sometime it takes a long time to load the page (depending on wifi speed and other factors) so I decided to add wait.